addons/web/static/src/legacy/js/views/kanban/kanban_record.js

```diff
@@ -683,7 +683,7 @@ var KanbanRecord = Widget.extend(WidgetAdapterMixin, {
             if (elem === event.currentTarget) {
                 ischild = false;
             }
-            var test_event = events && events.click && (events.click.length > 1 || events.click[0].namespace !== 'bs.tooltip');
+            var test_event = events && events.click && !elem.classList.contains('o_quick_editable') && (events.click.length > 1 || events.click[0].namespace !== 'bs.tooltip');
```

Comment:
Since the bug is in timesheets App only, so I think the fix should be in its modules and not in `web` one.
With this fix, I cannot be sure we will not have any regression or unexpected behaviour in another kanban views. 😕

Reply:
@xavierbol This is a generic issue for kanban records respective to the number field. If we have any number field displayed on kanban record and have a cursor on it and click, will not open the form view. For example: In CRM > expected revenue amount > set cursor over there and then click, will not open the form view.

Reply:
@hbh-odoo test case is missing, would you please add test case for the scenario you fixed.